snorkel/parser.py

```diff
@@ -223,7 +223,10 @@ def __init__(self, tok_whitespace=False, split_newline=False, parse_tree=False):
                         backoff_factor=0.1,
                         status_forcelist=[ 500, 502, 503, 504 ])
         self.requests_session.mount('http://', HTTPAdapter(max_retries=retries))
-    
+        
+        # Use this for recongizing PTB tokens
+        self.ptb_rgx = re.compile(r'-[A-Z]{2}B-')
```

Comment:
I'm slightly concerned that with weird text (like stuff with Chemicals, etc.) this regex might not be specific enough.  We can just list out the PTB tokens in the `PTB` dict and use this?